views/admin/edit_user.erb

```diff
@@ -166,64 +166,68 @@
-    </div>
-<% end %>
-
-<% if vehicles.count < 3 %>
-    <a href="/vehicle/add/?user_id=<%= user.id %>">
-        <button type="submit" class="dcf-mt-1 dcf-btn dcf-btn-primary" style="margin-top:20px !important;">Add Vehicle</button>
-    </a>
-<% end %>
+    <% if SS_ID != 8 %>
+        <h2 class="dcf-txt-h4 dcf-mt-6">Update Vehicle Information</h2>
+        <% if vehicles.count > 0 %>
+            <div>
+                <table class="dcf-w-100% dcf-table dcf-table-bordered dcf-table-responsive" aria-label="Vehicles List">
+                    <thead>
+                        <tr>
+                            <th scope="col">License Plate</th>
+                            <th scope="col">State</th>
+                            <th scope="col">Make</th>
+                            <th scope="col">Model</th>
+                            <th scope="col">Actions</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        <% vehicles.each do |vehicle| %>
+                            <tr>
+                                <td>
+                                    <%= vehicle.license_plate %><br>
+                                </td>
+                                <td>
+                                    <%= vehicle.state %>
+                                </td>
+                                <td>
+                                    <%= vehicle.make %>
+                                </td>
+                                <td>
+                                    <%= vehicle.model %>
+                                </td>
+                                <td class="table-actions">
+                                    <a href="/vehicle/<%= vehicle.id %>/edit/?user_id=<%= user.id %>" class="dcf-mt-1 dcf-btn dcf-btn-primary">Edit</a>
+                                    <form class="dcf-form delete-form delete-vehicle" action="/vehicle/<%= vehicle.id %>/delete/?user_id=<%= user.id %>" method="POST">
+                                        <button type="submit" class="dcf-mt-1 dcf-btn dcf-btn-primary">Delete</button>
+                                    </form>
+                                </td>
+                            </tr>
+                        <% end %>
+                    </tbody>
+                </table>
+            </div>
+        <% end %>
+    <% end %>
+    
+    <% if SS_ID != 8 %>
+        <% if vehicles.count < 3 %>
+            <a href="/vehicle/add/?user_id=<%= user.id %>">
+                <button type="submit" class="dcf-mt-1 dcf-btn dcf-btn-primary" style="margin-top:20px !important;">Add Vehicle</button>
+            </a>
+        <% end %>
+    <% end %>
 
-<% append_script_declaration("WDN.initializePlugin('datepickers');") %>
+    <% append_script_declaration("WDN.initializePlugin('datepickers');") %>
 
-<% append_script_declaration(%q[require(['jquery'], function($) {
+    <% append_script_declaration(%q[require(['jquery'], function($) {
```

Comment:
Move this script up inside the if statement since it has to do with the form inside the if statement